modules/rollup-plugin-workbox-inject/index.js

```diff
@@ -31,39 +33,74 @@ const chalk = require('chalk');
 function workboxInject(config) {
   return {
     name: 'workbox-inject', // this name will show up in warnings and errors
-    async generateBundle(opts, bundle) {
-      const options = validate(config, injectManifestSchema);
-      // Make sure we leave swSrc and swDest out of the precache manifest.
-      for (const file of [options.swSrc, options.swDest]) {
-        options.globIgnores.push(
-          rebasePath({
-            file,
+    async generateBundle(outputOptions, bundle) {
+      // injectionPoint isn't supported by getManifestSchema, but we need it.
+      const injectionPoint = config.injectionPoint || defaults.injectionPoint;
+      delete config.injectionPoint;
+
+      const options = validate(config, getManifestSchema);
+
+      // globDirectory might not be set, as developers can just provide
+      // additionalManifestEntries and forgo globbing.
+      if (options.globDirectory) {
```

Comment:
Bug fix: `globDirectory` might not always be set, and the previous version had problems if it wasn't.